code/modules/mob/mob.dm

```diff
@@ -42,6 +42,7 @@
 				hud_list[hud] = list()
 			else
 				var/image/I = image('icons/mob/hud.dmi', src, "")
+				I.appearance_flags = RESET_COLOR
```

Comment:
Might as well make this `I.appearance_flags = RESET_COLOR|RESET_TRANSFORM` to head off another problem before it becomes one.

Reply:
Wouldn't it make sense to not reset the transform if the mob is offset, though? e.g. lying in bed, etc.

Reply:
99% of the time, players read it from left to right---someone lying on the ground suddenly makes it read vertically--which is a bit strange---this info is typicalllyy something you want  presented to the player in the same way.

What benefit is there to the HUD looking upside down and backwards just because the mob's transform is set that way? Doesn't make much sense.

Reply:
Fair point, I'll add it in